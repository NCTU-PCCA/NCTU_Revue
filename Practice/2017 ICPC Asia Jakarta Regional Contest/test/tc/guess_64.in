166028004260286 48
><<<<<><<>><<<<><<<><<>>>>><><<>>>>>>>>><<<><>>=
