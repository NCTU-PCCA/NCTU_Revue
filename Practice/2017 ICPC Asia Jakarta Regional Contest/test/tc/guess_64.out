83014002130143 124521003195215 103767502662679 93390752396411 88202377263277 85608189696710 84311095913426 84959642805068 84635369359247 84473232636336 84554300997791 84594835178519 84574568088155 84564434542973 84559367770382 84556834384086 84558101077234 84557467730660 84557151057373 84556992720729 84557071889051 84557032304890 84557012512809 84557022408849 84557027356869 84557029830879 84557031067884 84557031686387 84557031377135 84557031531761 84557031454448 84557031415791 84557031435119 84557031444783 84557031449615 84557031452031 84557031453239 84557031453843 84557031454145 84557031454296 84557031454372 84557031454334 84557031454315 84557031454305 84557031454310 84557031454307 84557031454308 84557031454309
